corehq/apps/app_manager/views/apps.py

```diff
@@ -551,14 +552,29 @@ def app_exchange(request, domain):
     template = "app_manager/app_exchange.html"
     records = []
     for obj in ExchangeApplication.objects.all():
-        app = get_app(obj.domain, obj.app_id)
+        results = get_all_built_app_results(obj.domain, app_id=obj.app_id)
+        results = [r['value'] for r in results if r['value']['is_released']]
+        if not len(results):
```

Comment:
nit: i this is more pythonic to just say `if not results`

Reply:
99735fdf009